app/helpers/application_helper.rb

```diff
@@ -115,13 +117,11 @@ def role_allows?(**options)
       end
     end
 
-    Rbac.role_allows?(options.merge(:user => User.current_user)) rescue false
+    Rbac.role_allows?(:user => User.current_user, **options) rescue false
```

Comment:
rubocop does not like this line.

I will have to agree a little, it was blowing up due to an actual error and returning false.

Not sure if we want to fix at this time.

Reply:
currently, this is only passing `user`, `feature`, and `any`. But I'm leaving as `**options` b/c that seems to be your preference.

Reply:
I tracked it down to this PR's change:  https://github.com/ManageIQ/manageiq/pull/153/files#diff-bbb962ffbe850a018c5623bee8726489b9a4280ff05712b9560ef81529050eb7R52

I don't know if it's needed.  I wouldn't expect `role_allows?` to raise though.

Reply:
But, that's for another PR.  You're not changing behavior here.